src/cmd-koji-upload

```diff
@@ -429,12 +429,10 @@ class Reserve(_KojiBase):
         """
         log.info("Reserving a unique koji id")
 
-        release = datetime.datetime.utcnow().strftime("%H%M%S")
-
         data = {
             "name": f"{build.build_name}-{build.basearch}",
-            "release": release,
-            "version": f"{build.build_id.replace('-', '.')}",
+            "release": f"{build.build_id[-1:]}",
+            "version": f"{build.build_id[:-2]}",
```

Comment:
Let's update this to be a little more tolerant if something changes in the future.

For example.. Right now I don't anticipate us ever getting anything other than `-0` at the end of a RHCOS build ID, but maybe in the future that could change and we could theoretically get up to `-9`, `-10`, etc.

It should be simple to just handle that today if we split based on the `-`:

```
[dustymabe@media ~]$ python3
Python 3.11.4 (main, Jun  7 2023, 00:00:00) [GCC 12.3.1 20230508 (Red Hat 12.3.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> build_id='414.92.202307170903-0'
>>> version, release = build_id.split('-')
>>> print(version, release)
414.92.202307170903 0
```